app/src/main/res/xml/preferences.xml

```diff
@@ -72,6 +72,13 @@
             app:key="username_value"
             app:limit="31"
             app:title="@string/username" />
+        <emu.skyline.preference.IntegerListPreference
```

Comment:
You might want to also migrate `log_level` to this, we were sort of lazy initially to not create something like this and just parsed the integer instead.

PS: Do ensure that `skyline::Settings` parses it correctly as well as that could break when utilizing this

Reply:
Was already planning to make every integer setting use this preference in a following PR, I'll keep this in mind.